cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -101,47 +118,47 @@ func (s *VMTServer) Run(_ []string) error {
 		// Refer to Bug: https://vmturbo.atlassian.net/browse/OM-18139
 		pType = stitching.UUID
 	}
+
 	probeConfig := &probe.ProbeConfig{
 		CadvisorPort:          s.CAdvisorPort,
 		StitchingPropertyType: pType,
 	}
 
-	kubeConfig, err := clientcmd.BuildConfigFromFlags(s.Master, s.KubeConfig)
-	if err != nil {
-		glog.Errorf("Error getting kubeconfig:  %s", err)
-		return err
-	}
-	// This specifies the number and the max number of query per second to the api server.
-	kubeConfig.QPS = 20.0
-	kubeConfig.Burst = 30
+	return probeConfig
+}
 
-	kubeClient, err := client.New(kubeConfig)
-	if err != nil {
-		glog.Fatalf("Invalid API configuration: %v", err)
+// Run runs the specified VMTServer.  This should never exit.
+func (s *VMTServer) Run(_ []string) error {
+	if s.KubeConfig == "" && s.Master == "" {
+		glog.Warningf("Neither --kubeconfig nor --master was specified.  Using default API client.  This might not work.")
 	}
 
-	leaderElectionClient, err := clientset.NewForConfig(restclient.AddUserAgent(kubeConfig, "leader-election"))
-	if err != nil {
-		glog.Fatalf("Invalid API configuration: %v", err)
+	glog.V(3).Infof("Master is %s", s.Master)
```

Comment:
put this log inside a if, since we can only get master when `s.Master` is specified